scripts/update-acvm-workspace-versions.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
```

Comment:
This script looks in the root cargo.toml for all of the acvm-repo local packages, it then goes into their Cargo.toml files, extracts their version and then comes back to the root Cargo.toml file and modifies the version field to match